include/exadg/postprocessor/normal_flux_calculation.cpp

```diff
@@ -0,0 +1,179 @@
+/*  ______________________________________________________________________
+ *
+ *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
+ *
+ *  Copyright (C) 2021 by the ExaDG authors
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *  ______________________________________________________________________
+ */
+
+// C/C++
+#include <fstream>
+
+// ExaDG
+#include <exadg/postprocessor/normal_flux_calculation.h>
+#include <exadg/utilities/create_directories.h>
+
+namespace ExaDG
+{
+template<int dim, typename Number>
+NormalFluxCalculator<dim, Number>::NormalFluxCalculator(
+  dealii::MatrixFree<dim, Number> const & matrix_free_in,
+  unsigned int const                      dof_index_in,
+  unsigned int const                      quad_index_in,
+  NormalFluxCalculatorData const &        data_in,
+  MPI_Comm const &                        mpi_comm_in)
+  : matrix_free(matrix_free_in),
+    dof_index(dof_index_in),
+    quad_index(quad_index_in),
+    clear_files(true),
```

Comment:
Why not?
```suggestion
    data(data_in),
    clear_files(true),
```
And of course, remove the assignment below.

Reply:
yes